daffodil-cli/NOTICE-bin

```diff
@@ -0,0 +1,74 @@
+Apache Daffodil
+Copyright 2018 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+Based on source code originally developed by
+- The Univerisity of Illinois National Center for Supercomputing Applications (http://www.ncsa.illinois.edu/)
+- Tresys Technology (http://www.tresys.com/)
+- International Business Machines Corporation (http://www.ibm.com)
+
+The following NOTICE information applies to binary components distributed with this project:
+
+Apache Commons IO
```

Comment:
Generally speaking, we only include apache notices if they include more restrictive information.  So the Xerces one should be replicated.

Reply:
Did you mean, so *only* the Xerces one should be replicated, since it has extra stuff in the NOTICE? 

Doing some more verification, I noticed "xml-apis" jar and "xml-resolver" jar weren't referenced in either the LICENSE or NOTICE file. Both are Apache v2 licensed and part of Apache, so their NOTICE should be excluded unless it has anything extra like Xerces does, correct?

Reply:
Yes, only the xerces notice needs to be propagated, since the others are just boiler plate.